assets/src/edit-story/output/page.js

```diff
@@ -119,13 +119,15 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
         </amp-story-grid-layer>
       </StoryAnimation.Provider>
       {hasPageAttachment && (
-        <amp-story-page-attachment
+        <amp-story-page-outlink
```

Comment:
Make sure this tag and its attributes are added to the `KSES` PHP class

Reply:
Ah, good point 💡